src/components/PieChart.js

```diff
@@ -0,0 +1,161 @@
+
+import React from 'react';
+import Highcharts from 'highcharts';
+import drilldown from 'highcharts/modules/drilldown.js';
+import noDataToDisplay from 'highcharts/modules/no-data-to-display.js';
+
+function getCategoriesSeriesData(datas) {
+	datas = datas[0].data;
```

Comment:
In general, following ideas of immutability, purity, etc... it's good not to re-assign the value of a function argument. A lot of type-based languages/frameworks don't even allow this. The freedom that raw JS gives is qute fascinating.